bundles/org.openhab.binding.freebox/src/main/java/org/openhab/binding/freebox/internal/api/model/FreeboxFtthStatusResponse.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.freebox.internal.api.model;
+
+import org.openhab.binding.freebox.internal.api.FreeboxException;
+
+/**
+ * The {@link FreeboxFtthStatusResponse} is the Java class used to map the
+ * response of the connection Ftth status API
+ * https://dev.freebox.fr/sdk/os/connection/#
+ *
+ * @author Gaël L'hopital - Initial contribution
+ */
+public class FreeboxFtthStatusResponse extends FreeboxResponse<FreeboxFtthStatus> {
```

Comment:
What about nullness annotations in the new classes?

Reply:
This is a very old binding without nullness annotations.
And the class you point  is an object for JSON mapping, The nonnull annotation is not very appropriate for such case.

Reply:
Agreed.